### DISCUSSION TITLE
How to disable format code sections?

### DISCUSSION THREAD

**Discussion by cane-bt** [sql-formatter-org/sql-formatter#804] [Q&A] [answered]
Hopefully this is the place of to check regarding vscode extensions [Prettier SQL VSCode - Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=inferrinizzard.prettier-sql-vscode).

I've tried disabling the formatter on specific lines.
But the formatting still works.
Is this a known issue? Or is it fine?
Below is a screen capture of the file when it is saved.

https://github.com/user-attachments/assets/043a554c-dcc8-4c86-b7b4-dd646ecab82c

My settings.json is as follows.
```json
{
    "[sql]": {
        "editor.defaultFormatter": "inferrinizzard.prettier-sql-vscode"
    },
    "Prettier-SQL.expressionWidth": 120,
    "Prettier-SQL.keywordCase": "upper",
}
```
I'll add any other info you need.

**Comment by nene** [comment, marked as answer]
First, please read the FAQ about [Prettier SQL VSCode extension](https://github.com/sql-formatter-org/sql-formatter?tab=readme-ov-file#im-having-a-problem-with-prettier-sql-vscode-extension).

Second, no, there is no such feature that would enable/disable formatting on part of a file.

**Reply by cane-bt** [reply]
Thank you.
I've managed to solve my problem thanks to you!
